erpnext/accounts/doctype/payment_reconciliation/test_payment_reconciliation.py

```diff
@@ -1845,6 +1846,107 @@ def test_cr_note_payment_limit_filter(self):
 		self.assertEqual(len(pr.invoices), 1)
 		self.assertEqual(len(pr.payments), 1)
 
+	def test_reconciliation_on_closed_period_payment(self):
+		from erpnext.accounts.doctype.account.test_account import create_account
+
+		self.create_company()
```

Comment:
setup() method would've called create_company() and create_cost_center(). Not need to manually call on each test case.